docker/docker-compose.yml

```diff
@@ -11,11 +11,12 @@ services:
       - influxdb
     command: /app/coordinator -c /app/config.toml
     volumes:
-      - ${PWD}/../configs/docker-dev.toml:/app/config.toml
+      - ${PWD}/configs/docker-dev.toml:/app/config.toml
     networks:
       - xain-fl
+    environment:
```

Comment:
This is something that we need to update on the kubernetes setup as well, otherwise only errors are logged.

Reply:
Will do so!

Reply:
Done!